lib/tasks/publish_missed_scheduled_editions.rake

```diff
@@ -0,0 +1,11 @@
+namespace :editions do
+  desc "Publish editions missed in scheduled publishing"
+  task publish_missed_scheduled_editions: :environment do
+    scheduled_editions = Edition.scheduled_for_publishing.where(:publish_at.lt => Time.zone.now)
```

Comment:
I wonder whether it's worth doing something more [similar to Whitehall](https://github.com/alphagov/whitehall/blob/267d615c1fb84b31b353128ce7cee49a745a952c/lib/tasks/scheduled_publishing.rake#L60) here, where it checks the current queue, which would reduce the chance of trying to queue a job that is already queued (but hasn't been processed yet)?

Reply:
As sidekiq seems to be a culprit as for why those editions were not published, and for the reasons we were not able to diagnose (see details in Trello card)  I think it is safer to publish them imitating manual action.